packages/strapi-plugin-content-manager/admin/src/containers/ListPage/index.js

```diff
@@ -229,7 +231,8 @@ export class ListPage extends React.Component {
       history,
       listPage: { filters, params },
     } = this.props;
-    const searchEnd  = `&_sort=${params._sort}&source=${this.getSource()}${generateSearchFromFilters(filters)}`;
+    const _q = params._q !== '' ? `&_q=${params._q}` : '';
+    const searchEnd  = `&_sort=${params._sort}${_q}&source=${this.getSource()}${generateSearchFromFilters(filters)}`;
```

Comment:
I never saw that before but are we sure we need to use a string like that instead of an object to make it work correctly? I mean, is the history API supporting object as a parameter?

It's very complicated to read this part of code...